build/scripts/build-bazel-s390x.sh

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+# © Copyright IBM Corporation 2024
+#
+# Use of this source code is governed by a LICENSE_NAME-style
+# license that can be found in licenses/APACHE-v2.0.txt.
+# LICENSE: Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
This line isn't necessary because the license is placed in the file in the `licenses/` directory.